packages/babel-helper-validator-identifier/scripts/generate-identifier-regex.cjs

```diff
@@ -2,7 +2,7 @@
 
 // Always use the latest available version of Unicode!
 // https://tc39.github.io/ecma262/#sec-conformance
-const version = "15.1.0";
+const version = "16.0.0";
```

Comment:
The extension of this file is also changed to `.cjs` since the package's type is "module".